mteb/tasks/Retrieval/__init__.py

```diff
@@ -1,5 +1,28 @@
 from __future__ import annotations
 
+# TODO
```

Comment:
I suggest we make RTEB a completely separate task, e.g. `mteb/tasks/RTEB` rather than having it as a subtask of `Retrieval`.